FinderKeeper/ProjectSettings/TagManager.asset

```diff
@@ -3,7 +3,10 @@
 --- !u!78 &1
 TagManager:
   serializedVersion: 2
-  tags: []
```

Comment:
I don't see a "Player" tag here, yet you reference said tag in RotateWithLocationProvider

Reply:
I'm not sure how the TagManger is meant to show tags in the binary but it definitely is there. Won't have time to look into that further